link-rewriter-filter.gemspec

```diff
@@ -18,9 +15,10 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency 'html-pipeline', "~> 1.0"
+  spec.add_dependency 'html-pipeline', "~> 2.0"
+  spec.add_dependency 'github-markdown', "~> 0.6"
```

Comment:
I believe because `class LinkRewriterFilter < HTML::Pipeline::MarkdownFilter`. Without the dependencies:

```
➜  link-rewriter-filter git:(html-pipeline-2) ✗ script/cibuild
Missing dependency 'github-markdown' for MarkdownFilter. See README.md for details.
```